docs-source/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Rerunning introspection"
+date: 2020-06-04T08:14:51-05:00
+draft: true
+weight: 5
+description: "This document describes rerunning the introspector job in the Oracle WebLogic Server in Kubernetes environment."
+---
+
+
+This document describes _how and when_ to rerun the introspector job in the Oracle WebLogic Server in Kubernetes environment.
+
+#### Overview
+
+In order to manage a domain, the operator needs to examine the domain configuration in WebLogic Server. It does this by running an _introspector job_
+on the domain definition when the domain is first deployed in the operator. If the configuration is modified,
+the introspector job must be run again. The exact behavior will depend on the model chosen, and the detected change. Often the running
+domain can be updated without restarting any servers. Currently, this is not supported for 
+the Model in Image [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}).
+
+#### Common re-introspection scenarios
+
+This document describes what actions allow avoiding the restart of servers in a number of common re-introspection scenarios:
+
+* Modifying the WebLogic configuration for Domain in PV domains.
+* Changing the custom domain configuration overrides (also called situational configuration) for Domain in PV and Domain in Image domains
+
+### Use cases
+
+#### Supported dynamic configuration changes
+
+Changes to the Oracle WebLogic Server configuration may not require a full restart, depending on the domain home location 
+and the type of configuration change.
+
+* **Domain in PV:**
+For a domain home on PV, it is often possible to avoid a full restart when making a dynamic change.
+    * Dynamic configuration changes that do not require a full restart:
+      * Adding a cluster, server, or dynamic server
+      * Changing a dynamic property such as a server's connection timeout property
+    * Dynamic configuration changes that require shutting down or restarting servers.:  
```

Comment:
I see you changed to `Dynamic configuration changes that require shutting down or restarting servers.:` 

Is this more accurately 'Dynamic configuration changes require a full domain shutdown and restart:'?   

If not, please describe the conditions where a restart of a subset of the cluster is acceptable.

If yes, please update the wording.

Reply:
I am not at all familiar with the underlying WebLogic functionality. I suspect that I am the wrong one to be documenting this.